### PR TITLE
Refactor/feed on streams

### PR DESCRIPTION
**Pull request by roman-khomenko-dev**
Feed functionality works using streams

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ straightforward as possible.
 
 ### Changed
 - dependencies update
+- rewrite feed on streams
```
Comment (sergey-lukianov) [inline CHANGELOG.md:17]: This is for customer, not for devs

Should be more like `updating feed rendering w/ new mechanics`

File: lib/chat/log.ex [hunk 2]
```diff
@@ -83,7 +83,7 @@ defmodule Chat.Log do
   defp build(later, earlier) do
     {{@db_key, earlier, ""}, {@db_key, later, :binary.copy(<<255>>, 100)}}
     |> Db.list()
-    |> Enum.map(fn {{@db_key, _index, who}, data} -> {who, data} end)
+    |> Enum.map(fn {{@db_key, _index, who}, data} -> {UUID.uuid4(), who, data} end)
```
Comment (sergey-lukianov) [inline lib/chat/log.ex:86]: this serves a bad example for next dev

id should be the same if we will rerender an element, so lets make it hash of `Enigma.hash("#{db_key}:#{index}:#{who}") |> Base.encode16(case: :lower)`

File: lib/chat_web/live/main_live/page/feed.ex [hunk 3]
```diff
@@ -124,4 +142,34 @@ defmodule ChatWeb.MainLive.Page.Feed do
...
+    |> stream_batch_insert(:action_feed_list, list,
+      at: -1,
+      dom_id: &"item-#{elem(&1, 0)}"
+    )
+    |> assign_items_uuid(list)
+  end
+
+  defp assign_feed_stream(socket, list) do
+    socket
+    |> stream(:action_feed_list, list, dom_id: &"item-#{elem(&1, 0)}")
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/page/feed.ex:157]: it should be at least `feed-item`, item is too general

Comment (sergey-lukianov) [reply lib/chat_web/live/main_live/page/feed.ex:157]: i would create a function to convert item to `dom_id`

File: CHANGELOG.md (hunk 1 above)
Comment (roman-khomenko-dev) [reply CHANGELOG.md]: Right, thanks

File: lib/chat/log.ex (hunk 2 above)
Comment (roman-khomenko-dev) [reply lib/chat/log.ex]: changed

File: lib/chat_web/live/main_live/page/feed.ex (hunk 3 above)
Comment (roman-khomenko-dev) [reply lib/chat_web/live/main_live/page/feed.ex]: name changed and use new function `item_dom_id/1`

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**